### DISCUSSION TITLE
BelongsTo with foreign_key not displayed correctly in edit/new form

### DISCUSSION THREAD

**Discussion by fynsta** [activeadmin/activeadmin#7889] [Help]
### Expected behavior

Row title for attribute with associated `belongs_to` should be the name of the relation (target class), row should be marked as required.

### Actual behavior

Row title is just the humanized attribute name, row is not marked as required.

### How to reproduce

Create two classes. Class1 belongs to Class2 other via some foreign key. Look at the generated edit form for Class1 by ActiveAdmin. Notice that the column associated with the foreign_key is displayed incorrectly.

Interestingly, in the show and index pages, the attribute name is properly replaced by the class name, so the mishandling only seems to happen in forms

**Comment by fynsta** [comment]
I don't understand, is this expected behaviour?

**Comment by javierjulio** [comment]
I don't know. I don't use the belongs_to feature so I don't plan to address anything for it. Since I realize you're asking a general question, I'll reopen and move to Discussions as a more appropriate place for this. Hopefully others that use that feature can chime in. Thank you.
